CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Add setting that allows switching the internal Sentry database location on Windows/Linux ([#616](https://github.com/getsentry/sentry-unreal/pull/616))
```

Comment:
Can we make the entry a bit more specific what options are available?
I.e. The config panel now supports the option to choose between X and Y?

Reply:
Good point, I've rephrased it to make things more explicit